CI failing

@felipecrs you have any idea?
Haven't made a release yet so that's still fine.

```
#22 208.1 http://downloads.asterisk.org/pub/telephony/sounds/releases/asterisk-core-sounds-en-ulaw-1.6.1.tar.gz
#22 208.2 
#22 208.2 gzip: stdin: not in gzip format
#22 208.2 tar: This does not look like a tar archive
#22 208.2 tar: Exiting with failure status due to previous errors
#22 208.2 make[1]: *** [Makefile:142: .asterisk-core-sounds-en-ulaw-1.6.1] Error 2
#22 208.2 make[1]: Leaving directory '/usr/src/asterisk/sounds'
#22 208.2 make: *** [Makefile:465: datafiles] Error 2
#22 ERROR: process "/bin/bash -euxo pipefail -c menuselect/menuselect --disable BUILD_NATIVE menuselect.makeopts;     JOBS=$(( $(nproc) + $(nproc) / 2 ));     make -j ${JOBS} all;     DESTDIR=\"${INSTALL_DIR}\" make install;     DESTDIR=\"${INSTALL_DIR}\" make samples" did not complete successfully: exit code: 2
------
 > [build 14/26] RUN     menuselect/menuselect --disable BUILD_NATIVE menuselect.makeopts;     JOBS=$(( $(nproc) + $(nproc) / 2 ));     make -j ${JOBS} all;     DESTDIR="/opt/asterisk" make install;     DESTDIR="/opt/asterisk" make samples:
#22 207.8 make[1]: Entering directory '/usr/src/asterisk/sounds'
################################# 100.0%
#22 208.1 http://downloads.asterisk.org/pub/telephony/sounds/releases/asterisk-core-sounds-en-ulaw-1.6.1.tar.gz
#22 208.2 
#22 208.2 gzip: stdin: not in gzip format
#22 208.2 tar: This does not look like a tar archive
#22 208.2 tar: Exiting with failure status due to previous errors
#22 208.2 make[1]: *** [Makefile:142: .asterisk-core-sounds-en-ulaw-1.6.1] Error 2
#22 208.2 make[1]: Leaving directory '/usr/src/asterisk/sounds'
#22 208.2 make: *** [Makefile:465: datafiles] Error 2
```

Guessing the sounds download changed or something?

--- a/asterisk/Dockerfile
+++ b/asterisk/Dockerfile
@@ -143,6 +143,9 @@ WORKDIR /usr/src/asterisk
 # Increase the number of format entries in SDPs to 128
 RUN patch -p1 < /usr/src/asterisk-patches/increase_max_number_of_sdp_format_entries.patch
 
+# Temporary patch to fix upstream bug
+RUN patch -p1 < /usr/src/asterisk-patches/follow_redirects.patch
+
 RUN \
     # Recreate the configure script as we patched it above for the new formats \
     ./bootstrap.sh; \

--- a/asterisk/patches/asterisk/follow_redirects.patch
+++ b/asterisk/patches/asterisk/follow_redirects.patch
@@ -1,0 +1,46 @@
+From 54b3aee5a31b754ffef493424048dc7f6b52abaf Mon Sep 17 00:00:00 2001
+From: Miguel Angel Nubla <miguelangel.nubla@gmail.com>
+Date: Thu, 1 Jun 2023 02:09:39 +0200
+Subject: [PATCH 1/2] Makefile downloader enable follow redirects
+
+---
+ configure.ac | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/configure.ac b/configure.ac
+index db76aad6c68..0a542e7d88c 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -296,7 +296,7 @@ if test "x${WGET}" != "x:"; then
+   DOWNLOAD_TO_STDOUT="${WGET} -q -O-"
+   DOWNLOAD_TIMEOUT='--timeout=$1'
+ elif test "x${CURL}" != "x:"; then
+-  DOWNLOAD="${CURL} -O --progress-bar -w \"%{url_effective}\n\""
++  DOWNLOAD="${CURL} -L -O --progress-bar -w \"%{url_effective}\n\""
+   DOWNLOAD_TO_STDOUT="${CURL} -Ls"
+   DOWNLOAD_TIMEOUT='--max-time $(or $2,$1)'
+ else
+
+From 1c8e6543a4614ab6ee6c75efb21db1d35f234e47 Mon Sep 17 00:00:00 2001
+From: Miguel Angel Nubla <miguelangel.nubla@gmail.com>
+Date: Thu, 1 Jun 2023 02:27:45 +0200
+Subject: [PATCH 2/2] Enable follow redirect for wget downloader in
+ configure.ac
+
+---
+ configure.ac | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/configure.ac b/configure.ac
+index 0a542e7d88c..bd8add5e4dd 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -292,7 +292,7 @@ DOWNLOAD=":"
+ DOWNLOAD_TO_STDOUT=
+ DOWNLOAD_TIMEOUT=
+ if test "x${WGET}" != "x:"; then
+-  DOWNLOAD=${WGET}
++  DOWNLOAD="${WGET} -L"
+   DOWNLOAD_TO_STDOUT="${WGET} -q -O-"
+   DOWNLOAD_TIMEOUT='--timeout=$1'
+ elif test "x${CURL}" != "x:"; then

